subject_files_status_for_dpdash2.py

```diff
@@ -70,6 +105,12 @@ def get_mri_status():
         except (KeyError,TypeError):
             pass
 
+    if test:
+        df_tmp = df_subject_scan.reset_index()[
+                ['entry_date', 'mri_data_exist',
+                          'file_name', 'session_num', 'network', 'file_loc']]
+        df_tmp['file_loc'] = df_tmp['file_loc'].fillna('zip')
+        print(df_tmp)
```

Comment:
This block breaks symmetry with other data types. So I suggest it to be present in Kevin's personal branch only.

Reply:
How about this? https://github.com/AMP-SCZ/utility/pull/107

Reply:
No that's not what I meant. We do not have such test in other data types e.g. EEG, AVL, survey, etc.